### PR TITLE
Fix bug causing missing studyspaces

### LINKED ISSUE #22: New studyspaces not appearing
New studyspaces, e.g. UCL Student Centre (id=58) are not appearing.

### PR DESCRIPTION
**Pull request by hueyy**
There was no provision for a change (addition/removal) of a studyspace. This PR fixes that, and also takes the opportunity to streamline the fetching & storing of studyspace data in redux.

Instead of only fetching /summary data for workspaces already present in redux, the app will now fetch and store all workspaces from /workspaces/sensors/summary (i.e. will no longer specify `survey_ids`). If a studyspace is not present in fetched data (i.e. it has been removed), then it will be removed from the redux store and will no longer be shown on the app.

Plural actions, i.e. `setIsFetchingSeatInfos`, `fetchSeatInfosSuccess`, and `fetchSeatInfosFailure` replace their singular predecessors. Instead of dispatching an action for every individual studyspace to set its success/failure fetching status, the plural actions will do it en masse. This is neater and slightly quicker.

Also refactored `capacity` to `total` so that it is in line with the data sent from uclapi. It was unnecessary and slightly confusing to rename `total` => `capacity`.

TLDR: uclapi is the single source of truth and studyspace data fetched from the server will replace studyspace data in redux. No more missing studyspaces. Fixed bug where error message might show up even though fetching of data was successful.

Resolves #22

### CODE REVIEW

**Comment by ChristopherHammond13** [comment]
Is this done and ready for CR now? 😍

**Comment by hueyy** [comment]
Yep, it's ready 😊
